openedx/core/djangoapps/schedules/management/commands/__init__.py

```diff
@@ -23,20 +23,27 @@ def add_arguments(self, parser):
         parser.add_argument('site_domain_name')
 
     def handle(self, *args, **options):
-        resolver = self.make_resolver(*args, **options)
-        self.send_emails(resolver, *args, **options)
+        self.log_debug('Args = %r', options)
```

Comment:
hmm.. why do we have this helper?  won't it hide the correct line location of the logging call?

Reply:
Yeah, I don't really like this helper. I think the idea was to give messages a consistent prefix, but I think (as per a TODO) that an adapter would be better.